pkg/jsonmessage/jsonmessage_test.go

```diff
@@ -23,16 +30,181 @@ func TestProgress(t *testing.T) {
 		t.Fatalf("Expected %q, got %q", expected, jp2.String())
 	}
 
+	expectedStart := "[==========>                                        ]     20 B/100 B"
+	jp3 := JSONProgress{Current: 20, Total: 100, Start: time.Now().Unix()}
+	// Just look at the start of the string
+	// (the remaining time is really hard to test -_-)
+	if jp3.String()[:len(expectedStart)] != expectedStart {
+		t.Fatalf("Expected to start with %q, got %q", expectedStart, jp3.String())
+	}
+
 	expected = "[=========================>                         ]     50 B/100 B"
-	jp3 := JSONProgress{Current: 50, Total: 100}
-	if jp3.String() != expected {
-		t.Fatalf("Expected %q, got %q", expected, jp3.String())
+	jp4 := JSONProgress{Current: 50, Total: 100}
+	if jp4.String() != expected {
+		t.Fatalf("Expected %q, got %q", expected, jp4.String())
 	}
 
 	// this number can't be negetive gh#7136
```

Comment:
Not something you introduced but s/negetive/negative/  - just to avoid a one-line PR in the future :-)